packages/synapse-react-client/src/assets/icons/Checkbox/Checked.tsx

```diff
@@ -0,0 +1,9 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
+import React from 'react'
+import { ReactComponent as CheckedIcon } from './Checked.svg'
+
+function Checked(props: SvgIconProps) {
```

Comment:
MUI checkbox uses SVGs for styling. So I pulled our checkbox styles from Figma and added them as icons.

I recently learned you can separate the SVG from the component and import using SVGR like this, so we can view the SVG in GitHub/IDE and still use it as a MUI Icon component.